src/app/basket/page.tsx

```diff
@@ -0,0 +1,52 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+"use client";
+
+import PageHeading from "@/components/PageHeading";
+import DatasetList from "@/components/datasetList";
+import Button from "@/components/button";
+import { faPaperPlane, faPlusCircle } from "@fortawesome/free-solid-svg-icons";
+import { useDatasetBasket } from "@/providers/DatasetBasketProvider";
+
+export default function Page() {
+  const { basket } = useDatasetBasket();
+
+  const heading = `Your Basket (${basket.length})`;
+
+  return (
+    <div className="w-full p-10">
+      <PageHeading>{heading}</PageHeading>
```

Comment:
Can we add a condition so that this heading appears only when we have >=1 datasets?

(Maybe later on we can display the number of the datasets if hovering over the Basket image on the header of the page wdyt?)

Reply:
Acutally we already showing the number in the header. The screenshots are not up-to-date 

Here are the current ones

<img width="978" alt="Screenshot 2024-03-05 at 16 45 03" src="https://github.com/GenomicDataInfrastructure/gdi-userportal-frontend/assets/1210046/eec8620d-210b-4e67-9be0-bfcbfc66567a">

Regarding removing the heading, I can maybe remove `(0)` from the heading, but to remove complete heading, i will prefer less because almost all our page always have a heading 

what do you think?

Reply:
Yeah, if you remove the number should be fineee!